scripts/ia_healthcheck.py

```diff
@@ -0,0 +1,92 @@
+from datetime import datetime, timedelta
+import random
+from ..web_monitoring import db
+from ..web_monitoring import internetarchive
+
+
+MAX_CAPTURE_AGE = timedelta(hours=72)
+LINKS_TO_CHECK = 5
+
+
+def query_webmondb():
+    """
+    Query the web monitoring SCANNER API to get random links from the database
+    Interger -> List
+    """
+    try:
+        response = db.Client.from_env()
+    except Exception as e:
+        print('Could not access Web Monitoring API')
+        print(e)
+        return []
+    else:
+        page = response.list_pages(chunk=1, chunk_size=1)
+        url_count = page['meta']['total_results']
+        return ([response.list_pages(chunk=number, chunk_size=1)
+            ['data'][0]['url']
+            for number in random.sample(range(url_count), LINKS_TO_CHECK)])
+
+
+def query_waybackCDX(url):
+    """
+    Query Wayback Machine using CDX API
+    Get a list of versions of a url, and check for ValueError.
+    String -> JSON
+    """
+    try:
+        verisons = internetarchive.list_versions (url, from_date=datetime.now()
+            - MAX_CAPTURE_AGE)
+        verision = next(verisons)
+    except ValueError as e:
+        print(e)
+        status = False
+        return status
+    except:
+        print("Could not access Wayback CDX API")
```

Comment:
Might be useful to re-reaise the error so you get a stack trace in this case. This error would necessarily be something unexpected, so you’d probably want that, right?